distribution/test/itests/test-itests-dependencies-app/pom.xml

```diff
@@ -79,6 +79,28 @@
             <artifactId>test-itests-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.codice.imaging.nitf</groupId>
+            <artifactId>codice-imaging-nitf-core</artifactId>
+            <version>0.5</version>
```

Comment:
This version property exists in the imaging pom here :  https://github.com/codice/alliance/blob/master/catalog/imaging/pom.xml#L31.  might be worthwhile to move it to the root pom so it can be used everywhere.